Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -148,6 +154,8 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
     }()
 
     var pocketStories: [PocketStory] = []
+    var isRecentlySavedSectionHidden = false
+    var isRecentlySavedSectionEnabled = AppConstants.IS_RECENTLY_SAVED_SECTION_ENABLED
```

Comment:
This variable smacks of random middleman that we have to be careful when maintaining. I see you change it's setting later in the code. Could that all be done as one computed property that we don't have to worry about following? Maybe with an if/else based on whether it exists in profile prefs?